### PR TITLE
AT_US_17.02.001 | <Organization Folder> Rename

### PR DESCRIPTION
**Pull request by AramH20**
US - https://trello.com/c/aNpQHzNg/33-us1702-organization-folder-rename-organization-folder

TC - https://trello.com/c/kLyjvkGc/434-tc1702001-organization-folder-rename-organization-folder

TA - https://trello.com/c/8PeXN3re/440-atus1702001-organization-folder-rename-organization-folder

### CODE REVIEW

**Review by verafes — APPROVED** [review]

File: cypress/e2e/OrganizationFolderRename.cy.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+/// <reference types="cypress"/>
+
+import organizationFolderNames  from "../fixtures/organizationFolderNames.json";
+
+describe('Rename existing Organization Folder', () => {
+    beforeEach(() => {
+        cy.get('a[href$="/newJob"]').click();
+        // cy.get('.jenkins-input').as('nameInputField');
```
Comment (Rom-T) [inline cypress/e2e/OrganizationFolderRename.cy.js:8]: Please remove comment

File: cypress/e2e/OrganizationFolderRename.cy.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        cy.get('button[name=Submit]').click();
+        cy.get('[class="model-link"]').contains('Dashboard').click();
+    })
+
+    it('Rename Organization Folder using dropdown menu_positive', ()=> {
+        cy.get('a[href^="job/"').realHover();
+        cy.get('td > a [class$="dropdown-chevron"]').click();
+        cy.get('li > a > span').contains('Rename').click();
+        cy.get('.jenkins-input').clear();
+        cy.get('.jenkins-input').type(organizationFolderNames.newOrganizationFolder);
```
Comment (Rom-T) [inline cypress/e2e/OrganizationFolderRename.cy.js:22]: These 2 lines can be combined:
cy.get('.jenkins-input').clear().type(organizationFolderNames.newOrganizationFolder);

**Review by AramH20 — COMMENTED** [review]
Update test

**Review by Rom-T — APPROVED** [review]
LGTM

**Review by SvetlanaG1011 — APPROVED** [review]

**Pull request merged**